dex/src/main/scala/com/wavesplatform/dex/tool/ConfigChecker.scala

```diff
@@ -0,0 +1,205 @@
+package com.wavesplatform.dex.tool
+
+import cats.Eval
+import cats.data.NonEmptyList
+import com.typesafe.config.ConfigFactory.parseFile
+import com.typesafe.config._
+import com.wavesplatform.dex.settings._
+import com.wavesplatform.dex.cli.ErrorOr
+import com.wavesplatform.dex.error.Implicits.ThrowableOps
+import cats.syntax.either._
+import com.wavesplatform.dex.db.AccountStorage
+import com.wavesplatform.dex.domain.account.PublicKey
+import com.wavesplatform.dex.domain.asset.{Asset, AssetPair}
+import com.wavesplatform.dex.domain.bytes.ByteStr
+import com.wavesplatform.dex.settings.OrderFeeSettings._
+import pureconfig.{ConfigSource, ConfigWriter}
+import pureconfig._
+import pureconfig.generic.auto._
+import pureconfig.configurable.genericMapWriter
+import pureconfig.generic.semiauto
+import pureconfig.module.cats.nonEmptyListWriter
+import sttp.model.Uri
+
+import scala.jdk.CollectionConverters._
+import java.io.File
+import scala.util.Try
+
+object ConfigChecker extends ConfigWriters {
+
+  val skippedPaths = Seq("events-queue.kafka")
```

Comment:
It'd better to skip concrete paths those related to Java Kafka Clients:
* `events-queue.kafka.consumer.client`
* `events-queue.kafka.producer.client`

, because we have our settings in `events-queue.kafka`